packages/newspringchurchapp/src/prayer/index.js

```diff
@@ -35,15 +35,15 @@ const loadingStateObject = {
 // TODO: remove once this is pulling data
 const PrayerPreviewCardComponent = ({ route }) => (
   <PrayerPreviewCard
-    imageSource={{
+    avatarSource={{
       uri: 'https://fillmurray.com/400/600',
     }}
     name={'Bill'}
     overlayColor={['#FFF', '#FFF']}
     prayer={
       'I’m alright. Nobody worry ’bout me. Why you got to gimme a fight? Can’t you just let it be?'
     }
-    source={'Anderson'}
+    campus={'Anderson'}
```

Comment:
Great change

Reply:
@IsaacHardy @redreceipt This won't always be campus though, right? It could be "small group" or something else?